### PR TITLE
add ansible

### PR DESCRIPTION
**Pull request by HarrisChu**
add ansible-playbook

### CODE REVIEW

**Comment by wey-gu** [comment]
could we consider creating a dedecated repo and release for ansible codes?

File: ansible/templates/nebula-storaged.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,103 @@
+########## basics ##########
```
[the remark this answers is not in the document]
Comment (HarrisChu) [reply ansible/templates/nebula-storaged.conf.j2:1]: just copy from `etc/nebula-storaged.conf.production`.

add it

**Comment by HarrisChu** [comment]
> could we consider creating a dedicated repo and release for ansible codes?

agreed. @yixinglu how about using a dedicated repo

**Comment by yixinglu** [comment]
> > could we consider creating a dedicated repo and release for ansible codes?
> 
> agreed. @yixinglu how about using a dedicated repo

Good idea!

**Comment by HarrisChu** [comment]
transfer to nebula-ansible repo.
https://github.com/vesoft-inc/nebula-ansible

**Pull request closed without merging**